config/resources/newsletter-domain.json

```diff
@@ -0,0 +1,152 @@
+{
+  "version": "0.1",
+  "prefixes": {
+    "besluitvorming": "http://data.vlaanderen.be/ns/besluitvorming#",
+    "ext": "http://mu.semte.ch/vocabularies/ext/",
+    "dct": "http://purl.org/dc/terms/",
+    "nie": "http://www.semanticdesktop.org/ontologies/2007/01/19/nie#",
+    "owl": "http://www.w3.org/2002/07/owl#",
+    "prov": "http://www.w3.org/ns/prov#",
+    "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+    "skos": "http://www.w3.org/2004/02/skos/core#"
+  },
+  "resources": {
+    "news-items": {
+      "name": "news-item",
+      "class": "ext:Nieuwsbericht",
+      "attributes": {
+        "title": {
+          "type": "string",
+          "predicate": "dct:title"
+        },
+        "subtitle": {
+          "type": "string",
+          "predicate": "dct:alternative"
+        },
+        "html-content": {
+          "type": "string",
+          "predicate": "nie:htmlContent"
+        },
+        "plain-text": {
+          "type": "string",
+          "predicate": "prov:value"
+        },
+        "publication-date": {
+          "type": "datetime",
+          "predicate": "dct:issued"
+        },
+        "remark": {
+          "type": "string",
+          "predicate": "rdfs:comment"
+        },
+        "modified": {
+          "type": "datetime",
+          "predicate": "dct:modified"
+        },
+        "finished": {
+          "type": "boolean",
+          "predicate": "ext:afgewerkt"
+        },
+        "in-newsletter": {
+          "type": "boolean",
+          "predicate": "ext:inNieuwsbrief"
+        }
+      },
+      "relationships": {
+        "modified-by": {
+          "target": "user",
+          "predicate": "ext:modifiedBy",
+          "cardinality": "one"
+        },
+        "themes": {
+          "target": "theme",
+          "predicate": "dct:subject",
+          "cardinality": "many"
+        },
+        "agenda-item-treatments": {
+          "target": "agenda-item-treatment",
+          "predicate": "prov:wasDerivedFrom",
+          "cardinality": "many"
```

Comment:
See discussion in https://github.com/kanselarij-vlaanderen/frontend-kaleidos/pull/1561#discussion_r1046195169. Must be a `"one"` relation.